### PR TITLE
test: tracker integration dummy

### PR DESCRIPTION
**Pull request by axe312ger**

### CODE REVIEW

File: cypress/support/commands.ts [hunk 1]
```diff
@@ -23,3 +23,13 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('toggleIntegration', integrationLabel => {
```
Comment (zcei) [inline cypress/support/commands.ts:27]: 🤯 🥰

File: example/integrations/tracker-red-box-ltd.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+export function createRedBoxLtdIntegration(): IntegrationConfig {
+  return {
+    id: 'red-box-ltd',
+    title: 'Red Box Ltd.',
+    category: 'statistics',
+    description:
+      'Adds red borders around your content, demonstrates use of components that do e.g. click tracking',
+    color: '#C21515',
+    contrastColor: '#fff',
+    // @todo start react components with lower or upper case?
```
Comment (zcei) [inline example/integrations/tracker-red-box-ltd.tsx:94]: Checked react-router packages and formik for types and couln't really find consensus.
To not bloat the PR any further I'd leave this up to a follow-up PR and just keep as is. We'll likely revisit the exact shape of the config either way

**Review by zcei — APPROVED** [review]
This testing reads sooo well 👌

File: example/integrations/tracker-red-box-ltd.tsx (hunk 2 above)
Comment (axe312ger) [reply example/integrations/tracker-red-box-ltd.tsx:94]: exactly

**Pull request merged**